linux-create-vm.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Install git
+  echo "git should be installed"
+if ( which git )
+then 
+  echo "git already installed"
+else 
+  echo "installing git"
+  sudo apt install git
```

Comment:
This will sit and wait for input of "are you sure?" if you don't add a flag for yes:

`sudp apt install -y git`

Reply:
doesn't seem to be generating a prompt on a fresh multipass ubuntu vm